### ISSUE TITLE
 Code  Issues 184  Pull requests 0  Projects 0  Wiki Insights  Edit New issue Stacked Convolutional Auto-Encoders for Hierarchical Feature Extraction

### ISSUE THREAD

**Issue by leo-p** [leo-p/papers#25]
https://pdfs.semanticscholar.org/1c6d/990c80e60aa0b0059415444cdf94b3574f0f.pdf

We present a novel convolutional auto-encoder (CAE) for unsupervised feature learning. A stack of CAEs forms a convolutional neural network (CNN). Each CAE is trained using conventional on-line gradient descent without additional regularization terms. A max-pooling layer is essential to learn biologically plausible features consistent with those found by previous approaches. Initializing a CNN with filters of a trained CAE stack yields superior performance on a digit (MNIST) and an object recognition (CIFAR10) benchmark.

**Comment by leo-p** [comment]
## Summary:
- *Dataset:* [MNIST](yann.lecun.com/exdb/mnist/) and [SVHN](ufldl.stanford.edu/housenumbers/).
- *Objective:* Introduces the Convolutional Auto-Encoder, a hierarchical unsupervised feature extractor.

## Architecture:
Uses convolutions to generate an encoding of the image and then decodes it and do a pixel-wise comparison.
Used to initializes CNN.

## Results:
Old article, not really relevant nowadays. They don't speak about the deconvolution part.
